package.json

```diff
@@ -33,44 +33,44 @@
     "@aws-lambda-powertools/logger": "^2.7.0",
     "@aws-lambda-powertools/metrics": "^2.7.0",
     "@aws-lambda-powertools/tracer": "^2.7.0",
-    "@aws-sdk/client-dynamodb": "^3.629.0",
-    "@aws-sdk/client-sqs": "^3.629.0",
-    "@aws-sdk/util-dynamodb": "^3.629.0",
+    "@aws-sdk/client-dynamodb": "^3.645.0",
+    "@aws-sdk/client-sqs": "^3.645.0",
+    "@aws-sdk/util-dynamodb": "^3.645.0",
     "@smithy/node-http-handler": "^3.1.4"
   },
   "devDependencies": {
-    "@aws-sdk/client-sso-oidc": "^3.629.0",
-    "@aws-sdk/client-sts": "^3.629.0",
-    "@pact-foundation/pact": "^13.1.1",
+    "@aws-sdk/client-sso-oidc": "^3.645.0",
+    "@aws-sdk/client-sts": "^3.645.0",
+    "@pact-foundation/pact": "^13.1.2",
     "@pact-foundation/pact-node": "^10.18.0",
     "@stoplight/spectral-cli": "^6.11.1",
-    "@types/aws-lambda": "^8.10.143",
+    "@types/aws-lambda": "^8.10.145",
     "@types/express": "^4.17.21",
     "@types/jest": "^29.5.12",
-    "@types/node": "^20.14.15",
+    "@types/node": "^20.16.5",
     "@typescript-eslint/eslint-plugin": "^7.18.0",
     "@typescript-eslint/parser": "^7.18.0",
-    "aws-crt": "^1.21.3",
+    "aws-crt": "^1.22.0",
     "aws-lambda": "^1.0.7",
     "aws-sdk-client-mock": "^4.0.1",
     "aws-sdk-client-mock-jest": "^4.0.1",
-    "esbuild": "^0.23.0",
+    "esbuild": "^0.23.1",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-tsdoc": "^0.3.0",
     "eslint-plugin-unicorn": "^55.0.0",
     "express": "^4.19.2",
-    "husky": "^9.1.4",
+    "husky": "^9.1.5",
     "jest": "^29.7.0",
     "jest-extended": "^4.0.2",
     "jest-openapi": "^0.14.2",
     "prettier": "^3.3.3",
     "rimraf": "^6.0.1",
-    "ts-jest": "^29.2.4",
+    "ts-jest": "^29.2.5",
     "ts-node": "^10.9.2",
     "typescript": "^5.5.4",
-    "yarn-audit-fix": "^10.0.8"
+    "yarn-audit-fix": "^10.0.9"
   },
   "resolutions": {
```

Comment:
these resolution may not be needed anymore - please remove then do yarn install and audit to check